packages/grid/src/vaadin-grid-selection-column.d.ts

```diff
@@ -39,21 +40,10 @@ export interface GridSelectionColumnEventMap extends HTMLElementEventMap, GridSe
  *
  * __The default content can also be overridden__
  *
+ * @mixes GridSelectionColumnBaseMixin
```

Comment:
nit: we generally don't use `@mixes` in `.d.ts` files as it's only needed for `polymer-analyzer` that checks JS.